test/webpack-ts/webpack.config.js

```diff
@@ -1,9 +1,11 @@
-'use strict';
```

Comment:
what's the intention in removing strict mode? afaik it doesn't have any downside and only avoids errors.

Reply:
ES modules are strict by default, so there is no need for this expression.

Reply:
good to know, thanks :+1: